pkgs/development/python-modules/b2sdk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, mock, pyflakes, yapf, nose, arrow, logfury, requests, setuptools, six, tqdm}:
+
+buildPythonPackage rec {
+  pname = "b2sdk";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "143knykrlsdmk0748s4m7blp71r5n2bnk2gbsn72jlhn81zjyh0h";
+  };
+
+	postPatch = ''
+		sed 's/==\([0-9]\.\?\)\+//' -i requirements-test.txt
+	  substituteInPlace requirements-test.txt --replace "liccheck" ""
+  '';
+
+	checkInputs = [ mock pyflakes yapf nose ];
+	propagatedBuildInputs = [ arrow logfury requests setuptools six tqdm yapf ];
```

Comment:
```suggestion
	propagatedBuildInputs = [ arrow logfury requests setuptools six tqdm yapf ]
	  ++ lib.optionals isPy27 [ futures ];
```

```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/rich7lsi69jzm0nw84fnza1sr3dnizhb-python2.7-b2sdk-1.0.2.drv' failed with exit code 1; last 10 log lines:
  Executing pipInstallPhase
  /build/b2sdk-1.0.2/dist /build/b2sdk-1.0.2
  DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Processing ./b2sdk-1.0.2-py2-none-any.whl
  Requirement already satisfied: requests>=2.9.1 in /nix/store/rrxs8mkw2x96ia6jx2vakd3lrkb59fsz-python2.7-requests-2.22.0/lib/python2.7/site-packages (from b2sdk==1.0.2) (2.22.0)
  Requirement already satisfied: logfury>=0.1.2 in /nix/store/agq9y4w54cwnijylsp51ql64k5qr5jhs-python2.7-logfury-0.1.2/lib/python2.7/site-packages (from b2sdk==1.0.2) (0.1.2)
  Requirement already satisfied: tqdm>=4.5.0 in /nix/store/5ivwbb6zajiq63jdhm9spks5lkvjivkg-python2.7-tqdm-4.40.2/lib/python2.7/site-packages (from b2sdk==1.0.2) (4.40.2)
  Requirement already satisfied: setuptools in /nix/store/pfibnhl0w46pa2kyn47fdr5h8s466xh6-python2.7-setuptools-44.0.0/lib/python2.7/site-packages (from b2sdk==1.0.2) (44.0.0.post20200106)
  ERROR: Could not find a version that satisfies the requirement futures>=3.0.5; python_version < "3.2" (from b2sdk==1.0.2) (from versions: none)
  ERROR: No matching distribution found for futures>=3.0.5; python_version < "3.2" (from b2sdk==1.0.2)
cannot build derivation '/nix/store/dl7yczsczhrmhvw1s86c9gmyh308b3id-env.drv': 1 dependencies couldn't be built
```